persistence.xml not present while hot reloading

Hi !

When a Wisdom project is launched with wisdom:run and using a persistence.xml file, hot reloading always fail.

Logs output that the persistence.xml file is not present in the new bundle, even if the wisdom-jpa-enhancer correctly re-enhance entities.

I guess the persistence.xml file is included in the classic compile process but not in case of a reload ... but I didn't find where to fix that. Any idea where to look at ?

Thx !

```console
[INFO] 
[INFO] The watcher has detected a change in ...
[INFO] 
[INFO] Changes detected - recompiling the module!
[WARNING] File encoding has not been set, using platform encoding UTF-8, i.e. build is platform dependent!
[INFO] Compiling 20 source files to ...
902  waarheid  INFO   [wisdom-pipeline-watcher-pool-2-thread-1] openjpa.Tool - Enhancer running on type "...".
931  waarheid  WARN   [wisdom-pipeline-watcher-pool-2-thread-1] openjpa.Enhance - The type "..." must have a no-args constructor. Adding a public no-args constructor.
956  waarheid  INFO   [wisdom-pipeline-watcher-pool-2-thread-1] openjpa.Tool - Enhancer running on type "...".
968  waarheid  INFO   [wisdom-pipeline-watcher-pool-2-thread-1] openjpa.Tool - Enhancer running on type "...".
970  waarheid  WARN   [wisdom-pipeline-watcher-pool-2-thread-1] openjpa.Enhance - The type "..." must have a no-args constructor. Adding a public no-args constructor.
981  waarheid  INFO   [wisdom-pipeline-watcher-pool-2-thread-1] openjpa.Tool - Enhancer running on type "...".
982  waarheid  WARN   [wisdom-pipeline-watcher-pool-2-thread-1] openjpa.Enhance - The type "..." must have a no-args constructor. Adding a public no-args constructor.
991  waarheid  INFO   [wisdom-pipeline-watcher-pool-2-thread-1] openjpa.Tool - Enhancer running on type "...".
994  waarheid  WARN   [wisdom-pipeline-watcher-pool-2-thread-1] openjpa.Enhance - The type "...l" must have a no-args constructor. Adding a public no-args constructor.
1002  waarheid  INFO   [wisdom-pipeline-watcher-pool-2-thread-1] openjpa.Tool - Enhancer running on type "...".
[WARNING] Updating import version of com.google.common* to 19.0.0
[WARNING] Export ...,  has 1,  private references [...], 
[ERROR] Meta-Persistence refers to resources not in the bundle: [/META-INF/persistence.xml]
[ERROR] Meta-Persistence refers to resources not in the bundle: [/META-INF/persistence.xml]
I Apache Felix iPOJO Manipulator - 1.12.1
[INFO] Copying waarheid-0.1.0-SNAPSHOT.jar to ....jar
[INFO] 
[INFO] 
[INFO]	 o.o.c.c.a.DirectoryMonitor {monitor-application} - File ....jar from .../target/wisdom/application changed 
[INFO]	 o.o.c.c.a.BundleDeployer {monitor-application} - Updating bundle ... - ....jar 
[INFO]	 o.w.r.RequestRouter {monitor-application} - Removing routes from ...
[INFO]	 o.w.f.j.JPATransformer {monitor-application} - unregister transformers from bundle ... [64] 
[INFO]	 o.w.f.j.JPAManager {monitor-application} - META_PERSISTENCE header found in bundle 64 : /META-INF/persistence.xml 
[INFO]	 o.w.f.j.JPAManager {monitor-application} - Analysing location /META-INF/persistence.xml 
[INFO]	 o.w.f.j.JPAManager {monitor-application} - Parsed persistence: org.wisdom.framework.jpa.model.Persistence@ae912d, unit [org.wisdom.framework.jpa.model.Persistence$PersistenceUnit@155e71d] 
[INFO]	 o.w.f.j.JPAManager {monitor-application} - Recycling persistence unit org.wisdom.framework.jpa.model.Persistence$PersistenceUnit@7f74c213 
[WARN]	 o.w.f.j.JPAManager {monitor-application} - No persistence unit found in bundle 64, despite a META_PERSISTENCE header (/META-INF/persistence.xml) 
[INFO]	 o.w.f.j.JPAManager {monitor-application} - META_PERSISTENCE header found in bundle 64 : /META-INF/persistence.xml 
[INFO]	 o.w.f.j.JPAManager {monitor-application} - Analysing location /META-INF/persistence.xml 
[INFO]	 o.w.f.j.JPAManager {monitor-application} - Parsed persistence: org.wisdom.framework.jpa.model.Persistence@526658c, unit [org.wisdom.framework.jpa.model.Persistence$PersistenceUnit@1849d9e1] 
[INFO]	 o.w.f.j.JPAManager {monitor-application} - Recycling persistence unit org.wisdom.framework.jpa.model.Persistence$PersistenceUnit@7f74c213 
[WARN]	 o.w.f.j.JPAManager {monitor-application} - No persistence unit found in bundle 64, despite a META_PERSISTENCE header (/META-INF/persistence.xml) 
[INFO]	 o.w.r.RequestRouter {[iPOJO] pool-1-thread-1} - Adding routes from ...@77cc702d 
[INFO]	 o.w.r.RequestRouter {[iPOJO] pool-1-thread-1} - Adding routes from ...t@6d267454 


```

--- a/wisdom-jpa-manager/src/main/java/org/wisdom/framework/jpa/JPAManager.java
+++ b/wisdom-jpa-manager/src/main/java/org/wisdom/framework/jpa/JPAManager.java
@@ -200,6 +200,7 @@ public class JPAManager {
                     } else {
                         Persistence.PersistenceUnit previousPu = pus.get(jta);
                         previousPu.getClazz().addAll(pu.getClazz());
+                        set.add(previousPu);
                         LOGGER.info("Recycling persistence unit {}", previousPu);
                     }
                 }

